generators/entity-client/templates/react/src/main/webapp/app/entities/entity-update.tsx.ejs

```diff
@@ -89,7 +89,11 @@ Object.keys(differentRelationships).forEach(key => {
     if (uniqueRel.otherEntityAngularName === 'User') {
 _%>
 import { I<%= uniqueRel.otherEntityAngularName %> } from 'app/shared/model/user.model';
+<%_ if (authenticationType === 'oauth2') { _%>
```

Comment:
I think there is a stray file `user-service.spec.ts.ejs` in this PR. Can you check. Other than that, this PR is ready to merge